github/GitRelease.py

```diff
@@ -188,6 +189,9 @@ def update_release(
             "draft": draft,
             "prerelease": prerelease,
         }
+        if make_latest is not NotSet:
+            assert isinstance(make_latest, bool), make_latest
```

Comment:
documentation says, this is a string
```suggestion
            assert make_latest in ["true", "false", "legacy"], make_latest
```